privacy-on-beam/pbeam/mean_test.go

```diff
@@ -1155,3 +1154,77 @@ func TestMeanPerKeyWithPartitionsNoNoiseIntValues(t *testing.T) {
 		}
 	}
 }
+
+// Checks that MeanPerKey with empty specified partitions returns a correct answer for int input values.
+// They should be correctly converted to float64 and then correct result
+// with float statistic should be computed.
+func TestMeanPerKeyWithEmptyPartitionsNoNoise(t *testing.T) {
+	for _, tc := range []struct {
+		lower float64
+		upper float64
+	}{
+		// Used for MinValue and MaxValue. Tests case when specified partitions are already in the data.
+		{
+			lower: 1.0, 
+			upper: 3.0,
+		},
+		{
+			lower: 0.0,
+			upper: 2.0,
+		},
+		{
+			lower: -10.0,
+			upper: 10.0,
+		},
+	} {
+		triples := concatenateTriplesWithIntValue(
+			makeTripleWithIntValue(7, 0, 2))
+
+		midpoint := tc.lower + (tc.upper-tc.lower)/2.0
+		exactCount := 0.0
+		exactMean := midpoint
```

Comment:
Please add an inline comment along the lines of `Mean of 0 elements is midpoint.`